### DISCUSSION TITLE
v2.3.0

### DISCUSSION THREAD

**Discussion by keshav-space** [keshav-space/safenotes#148] [Announcements]
## What's Changed [TL;DR]
* Support passphrase autofill
* Support for 7 new languages: **Czech**, **German**, **Indonesian**, **Norwegian**, **Polish**, **Spanish** and **Ukrainian**
* Autorotation support [_experimental_]
* Target SDK version is now 33

## What's Changed [Details]
* German and Russian translation by @Atalanttore and @TheDarkKRONOS in https://github.com/keshav-space/safenotes/pull/115
* Polish translation by @rehork  in https://github.com/keshav-space/safenotes/pull/116
* Norwegian Bokmål and Spanish translation by @comradekingu and @Bost0n in https://github.com/keshav-space/safenotes/pull/121
* Spanish and Russian translation by @Bost0n and @TheDarkKRONOS in https://github.com/keshav-space/safenotes/pull/126
* Indonesian translation by @nature-modren in https://github.com/keshav-space/safenotes/pull/128
* Indonesian and Bengali translation by @nature-modren and @ABN-Anik in https://github.com/keshav-space/safenotes/pull/129
* Portuguese (Brazil) translation by @fnogcps in https://github.com/keshav-space/safenotes/pull/132
* Support Polish, German, Indonesian and Spanish by @keshav-space in https://github.com/keshav-space/safenotes/pull/133
* Czech translation by @pchelium in https://github.com/keshav-space/safenotes/pull/139
* Add autorotate setting by @keshav-space in https://github.com/keshav-space/safenotes/pull/140
* Add copyright header by @keshav-space in https://github.com/keshav-space/safenotes/pull/141
* Support passphrase autofill by @keshav-space in https://github.com/keshav-space/safenotes/pull/142
* Chinese (Simplified) and Italian translation by @TYEclipse and @Toredane in https://github.com/keshav-space/safenotes/pull/143
* Support Czech and Norwegian by @keshav-space in https://github.com/keshav-space/safenotes/pull/144
* Ukrainian translation by @Kefir2105 in https://github.com/keshav-space/safenotes/pull/145
* Bump targetSdkVersion by @keshav-space in https://github.com/keshav-space/safenotes/pull/146

## New Contributors
* @rehork made their first contribution in https://github.com/keshav-space/safenotes/pull/116
* @Bost0n made their first contribution in https://github.com/keshav-space/safenotes/pull/126
* @nature-modren made their first contribution in https://github.com/keshav-space/safenotes/pull/128
* @ABN-Anik made their first contribution in https://github.com/keshav-space/safenotes/pull/129
* @fnogcps made their first contribution in https://github.com/keshav-space/safenotes/pull/132
* @pchelium made their first contribution in https://github.com/keshav-space/safenotes/pull/139
* @TYEclipse  made their first contribution in https://github.com/keshav-space/safenotes/pull/143
* @Kefir2105 made their first contribution in https://github.com/keshav-space/safenotes/pull/145

**Full Changelog**: https://github.com/keshav-space/safenotes/compare/v2.2.0...v2.3.0

```
Note: If not sure of your device architecture use safenotes-2.3.0-all.apk
```

<hr /><em>This discussion was created from the release <a href='https://github.com/keshav-space/safenotes/releases/tag/v2.3.0'>v2.3.0</a>.</em>

**Comment by hallothere803** [comment]
Good job bro💪
